### PR TITLE
Feature add security context

### PR DESCRIPTION
**Pull request by obeyler**
Add a securityContext to be able to set an { runAsUser: 1001}
This will allow to deploy it in a K8S where the PodSecurityPolicy restricted as RunAsNonRoot

### CODE REVIEW

**Comment by obeyler** [comment]
Hello @drnic is there any trouble with this PR ?

File: helm/worlds-simplest-service-broker/Chart.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: World's Simplest Service Broker for Service Catlog
 name: worlds-simplest-service-broker
-version: 1.3.1
+version: 1.3.2
```
Comment (drnic) [inline helm/worlds-simplest-service-broker/Chart.yaml]: CI will bump this when it ships a release

**Review by drnic — CHANGES_REQUESTED** [review]

**Comment by drnic** [comment]
Thanks for PR. Sorry I didn’t review it much earlier.

File: helm/worlds-simplest-service-broker/Chart.yaml (hunk 1 above)
Comment (obeyler) [reply helm/worlds-simplest-service-broker/Chart.yaml]: Ok

Comment (obeyler) [reply helm/worlds-simplest-service-broker/Chart.yaml]: @drnic  I decrease the version number as requested

**Comment by drnic** [comment]
Oops, pressed the wrong button.

**Pull request merged**